webpack-common.config.js

```diff
@@ -44,7 +47,17 @@ async function getBaseConfig() {
                 },
                 {
                     test: /\.[tj]s$/,
-                    include: srcPath,
+                    include: coreSrcPath,
```

Comment:
Is there a way we can specify the `ts-loader` without having to do it individually for each package?

Reply:
I linked the ts-loader issue in the PR, we can try use tsconfig includes going forward but was not sure if we need to upgrade ts-loader for that. 
I am going to move this config to a new file so we can have all the config object in one place.

Another way is what we discussed about moving builds to each package, but that may require thinking around how to export types.